coding_style/README.adoc

```diff
@@ -23,10 +23,15 @@ Rationale:: even when Ansible currently allows names that are not valid identifi
 Making all names valid identifiers will avoid encountering problems in the future. Dictionary keys that are not valid identifiers are also less intuitive to use in Jinja2 (a dot in a dictionary key would be particularly confusing).
 ====
 
+* Use human-readable variable names
 * Use mnemonic and descriptive names and do not shorten more than necessary.
   A pattern `object[_feature]_action` has proven useful as it guarantees a proper sorting in the file system for roles and playbooks.
   Systems support long identifier names, so use them!
 * Avoid numbering roles and playbooks, you'll never know how they'll be used in the future.
+* All tasks should be named and start with a capital letter, this improves readability.
```

Comment:
I disagree with this one, I name my tasks with a non-capital letter, it looks IMHO better in the resulting log.

This said, I'm flabbergasted that I oversaw the rule to have all tasks named, so well catched.

Reply:
@ericzolf Do you consider this line a blocker for this PR?  I'm not clear if you have a differing opinion but don't feel strongly or if we need to remove this guidance.

Reply:
I'd be fine with "Name all tasks to improve readability of the logs", I've created a separate issue to discuss capital or non-capital names (we can also decide to _not_ make a recommendation).

Reply:
@ericzolf WDYT about opening threads under https://github.com/ansible/ansible-lint/discussion even as "polls" is we already know the possible directions or normal ideas if they directions are not yet clear. This will makes the debate/decision process considerably more inclusive?

In fact there are several checks we need to implement and decide on related to name content: capitalized first letter? should it be imperative? should it end with a dot or not? should we ensure that same task name does not repeat inside current file, current role, globally? I did talk with @cidrblock and we are likely going to try to re-use python docstring checks for this.